autofill-parser/src/main/java/com/github/androidpasswordstore/autofillparser/FeatureAndTrustDetection.kt

```diff
@@ -183,7 +184,6 @@ internal fun getBrowserAutofillSupportInfoIfTrusted(context: Context, appPackage
 
 private val FLAKY_BROWSERS =
   listOf(
-    "org.bromite.bromite",
     "org.ungoogled.chromium.stable",
```

Comment:
Seems like Ungoogle Chromium [is also on v89](https://github.com/ungoogled-software/ungoogled-chromium-android/releases/tag/89.0.4389.90-1-webview), we should probably promote this as well

Reply:
I verified that Kiwi doesn't have the Autofill fixes yet. Once it gets them (or we decide that it is not important enough to warrant a special message), we can finally drop the whole flaky classification.